openml-lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/LightGBMSWIG.java

```diff
@@ -190,9 +190,11 @@ private void initGetBoosterNumFeatures() throws LightGBMException {
     }
 
     /**
+     * Gets number of features in the model.
+     *
      * @return Number of features in the model (retrieved from model binary).
      */
-    int getBoosterNumFeatures() {
+    public int getBoosterNumFeatures() {
```

Comment:
LightGBMSWIG class is meant for internal operations alone. Please restore all the methods visibility to the original.